content/en/docs/tutorials/security/apparmor.md

```diff
@@ -163,6 +176,10 @@ kubectl exec <pod_name> -- cat /proc/1/attr/current
 k8s-apparmor-example-deny-write (enforce)
 ```
 
+## Ephemeral containers
+
+The `container.apparmor.security.beta.kubernetes.io/*` annotations don't apply to Ephemeral containers. When an ephemeral container runs in [privileged mode](/docs/concepts/workloads/pods/#privileged-mode-for-containers), it disables AppArmor on the container. Otherwise, the ephemeral container runs with the default runtime profile.
```

Comment:
First, I'd wrap and reword this; eg:
```markdown
The `container.apparmor.security.beta.kubernetes.io/*` annotations don't apply to
ephemeral containers. When an ephemeral container runs in
[privileged mode](/docs/concepts/workloads/pods/#privileged-mode-for-containers),
the kubelet does not apply any AppArmor restrictions to the ephemeral container.
Otherwise, the ephemeral container runs with the default runtime profile.
```

Second: is “_Otherwise, the ephemeral container runs with the default runtime profile_” always true? There might be some conditions, such as the node having AppArmor installed and active.

Reply:
Hi @sftim, thanks for your comment. I agree. 

Maybe we could phrase it:

```
 The `container.apparmor.security.beta.kubernetes.io/*` annotations don't apply to
 ephemeral containers. When an ephemeral container runs in
 [privileged mode](/docs/concepts/workloads/pods/#privileged-mode-for-containers),
 the kubelet does not apply any AppArmor restrictions to the ephemeral container.
 Otherwise, the kubelet will attempt to run the ephemeral container with the default runtime profile.
```

WDYT?

Reply:
Sure, that sounds fine.

Reply:
Thanks @sftim, updated.